plugin-debezium-mongodb/src/main/java/io/kestra/plugin/debezium/mongodb/Trigger.java

```diff
@@ -143,6 +143,15 @@ public class Trigger extends AbstractTrigger implements PollingTriggerInterface,
     @Builder.Default
     private Property<MongodbInterface.SnapshotMode> snapshotMode = Property.ofValue(SnapshotMode.INITIAL);
 
+    @Schema(
+        title = "How to commit the offsets to the KV Store.",
+        description = """
+            - ON_EACH_BATCH: after each batch of records consumed by this trigger, the offsets will be stored in the KV Store. This avoids any duplicated records being consumed but can be costly if many events are produced.
+            - ON_STOP: when this trigger is stopped or killed, the offsets will be stored in the KV Store. This avoid any un-necessary writes to the KV Store, but if the trigger is not stopped gracefully, the KV Store value may not be updated leading to duplicated records consumption."""
+    )
+    @Builder.Default
+    protected Property<io.kestra.plugin.debezium.AbstractDebeziumRealtimeTrigger.OffsetCommitMode> offsetsCommitMode = Property.ofValue(io.kestra.plugin.debezium.AbstractDebeziumRealtimeTrigger.OffsetCommitMode.ON_STOP);
```

Comment:
Could you remove the packages whenever it's possible?

```suggestion
    protected Property<AbstractDebeziumRealtimeTrigger.OffsetCommitMode> offsetsCommitMode = Property.ofValue(AbstractDebeziumRealtimeTrigger.OffsetCommitMode.ON_STOP);
```